UndertaleModLib/Decompiler/Disassembler.cs

```diff
@@ -34,18 +34,24 @@ public static string GenerateLocalVarDefinitions(this UndertaleCode code, IList<
 
             return sb.ToString();
         }
-
+        
         public static string Disassemble(this UndertaleCode code, IList<UndertaleVariable> vars, UndertaleCodeLocals locals)
         {
-            StringBuilder sb = new StringBuilder();
+            // This stringbuilder is shared with the code instruction's ToString methods.
```

Comment:
```suggestion
            // This `StringBuilder` is shared with the `UndertaleInstruction.ToString()` methods.
```

Reply:
Then:
```suggestion
            // This StringBuilder is shared with the ToString() methods of the code instruction.
```

Because you should use "'s" only for animated objects

Reply:
Ah whoops, this wasn't supposed to be possessive, it was supposed to be plural. Changing this around does make it more obvious, yeah.
Still gonna veto the brackets, method name + it being literally called "method" is enough.